about.html

```diff
@@ -0,0 +1,118 @@
+             <li><a href="">Program</a></li>
+             <li><a href="">Join</a></li>
+              <li><a href="">Sponsor</a></li>
+               <li><a href="">News</a></li>
+                <li class="red-border"><a href="">CC Campaign</a></li>
+            </ul>
+        </div>
+        </div>
+
+
+    <div class="home-page">
+      <div class="description">
+        <h4 class="heading2">"Hello, Tourist!"</h4>
+        <h1 class="home-heading">
+          Visit Oromia, Ethiopia Land of Origin and Heritages in 2023!
+        </h1>
+        <p class="info">
+          Oromia is a largest region in Ethiopia with potential natural
+          resources and cultural heritages. Visit Oromia where the sheer
+          diversity of the culture, people and landscape is sure to leave you
+          astounded.
+        </p>
+
+        <div class="contact-us">
+          <p class="info">
+            Please contact-us for further explanation about visit oromia @
+            <br /><span class="creative-heading">VisitOromia@2023</span>
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="creative-container">
+        <h2 class="creative-heading">
+          Creative Commons Visit Oromia Logo in 2023!
+        </h2>
+        <hr />
+        <p class="about-info info">
+          The logo of visit Oromia was developed by the group of software
+          developer  who were qualified by visit-oromia tourism commission. <br> After
+          careful consideration of the process of selection though online boot
+          voting  we are glad to announce that Roba as a winner for the oromia
+          tourism logo winner.
+        </p>
+      </div>
+    <div class="logo-about logo-container">
+      <img src="images/bg-visit.jpg" alt="" />
+    </div>
+    <div class="live-oromia">
+      <div class="see-people">
+        <h2 class="creative-heading">
+          See Previous Visit Oromia Gallery in 2010 and 2016
+        </h2>
+        <p>Look at the people from Land of Oromia, Ethiopia in the pasts</p>
+      </div>
+
+      <div class="live-cont">
+        <div class="live-container1">
+          <h2>2015</h2>
+          <p>Visit Oromia in 2016</p>
+        </div>
+        <div class="live-container2">
+          <h2>2010</h2>
+          <p>Visit Oromia in 2010</p>
+        </div>
+      </div>
+    </div>
+    <div class="hidden-partner partner" id="partner">
+      <div class="partner-hr">
+        <h2>Partner</h2>
+        <hr class="part" />
+      </div>
+
+      <div class="partner-img-about">
+        <img src="images/oromia-bank.png" alt="" />
+        <img src="images/awash.jpg" alt="" />
+        <img src="images/ethiopian-air.png" />
+        <img src="images/land eth.png" alt="" />
+        <img src="images/oda.jpg" alt="" />
+      </div>
+    </div>
+    <div class="creative-common" id="creative">
+      <img src="images/bg-visit.jpg" alt="" />
+      <h3>@2015 Creative Common, Oromia, Ethiopia</h3>
+    </div>
```

Comment:
- [ ] I would suggest you please use the footer tag for the footer it indicates better semantic HTML.